src/com/sun/ts/tests/jaspic/tssv/config/TSAuthConfigProviderServlet.java

```diff
@@ -41,7 +41,7 @@
  */
 public class TSAuthConfigProviderServlet
     implements jakarta.security.auth.message.config.AuthConfigProvider {
-  private static TSLogger logger = TSLogger.getTSLogger(JASPICData.LOGGER_NAME);
```

Comment:
+1 for removing the eager init so that the first need for logger will actually use the initializeTSLogger